### ISSUE TITLE
Возможна ли автоматическая смена раскладки в зависимости от языка текста вокруг курсора? 

### ISSUE THREAD

**Issue by Userbit** [lyokha/vim-xkbswitch#36]
Узнал из документации, что имеется возможность настройки автоматического переключения раскладки на основе указания синтаксических правил в переменной **g:XkbSwitchSyntaxRules**.

Но возник вопрос. 
Возможно ли автоматическое переключение языка раскладки, при входе режим вставки, в зависимости от языка самого текста, который окружает текущее положение курсора? 

Было бы удобно иметь уже включенной ту раскладку, которая соответствует текущему языку окружающего текста. Чтобы не затрачивать время на ручное переключение.

**Comment by lyokha** [comment]
Нет, такой поддержки нет. Несмотря на то, что принять решение о переключении русского/английского достаточно просто, поскольку символы в этих языках не пересекаются, в общем случае это довольно сложная эвристическая задача: представьте как вы будете переключаться между английским и французским, например: тут символы под курсором присутствуют в обоих языках, и придется анализировать слова, а то и целые предложения, так как международные слова тоже существуют, и в большом количестве.

**Comment by Userbit** [comment]
Может тогда реализовать подобную особенность поведения только для частного случая, русско-английской раскладки.
